test/bigtest/tests/search-test.js

```diff
@@ -28,4 +28,38 @@ describe('Search', () => {
       expect(searchInteractor.countResults).to.be.greaterThan(1);
     });
   });
+
+  describe('submit with the button', function () {
```

Comment:
Great to see a test using the button. Also, update the description above to indicate that the first test 'fill search field and submit with the ENTER key'